src/composables/themes/themes.model.ts

```diff
@@ -54,6 +54,14 @@ export type Metadata = Partial<{
 }>
 
 export type LayerType = 'WMS' | 'WFS' | 'WMTS' | 'BG WMTS' | 'BG MVT' | '3D'
```

Comment:
Maybe it would make sense to rename this `LayerTypeValue` to prevent the same name for the type and the const?

Reply:
yes, will correct that